ui-speedspacechart/src/__tests__/utils.spec.ts

```diff
@@ -351,8 +348,49 @@ describe('getStopPosition', () => {
     const width = 1000;
     const ratioX = 1;
     const maxPosition = 600;
-    const stopPosition = getStopPosition(position, width, ratioX, maxPosition);
-    expect(stopPosition).toBe(336);
+    const stopPosition = positionToPosX(position.start, maxPosition, width, ratioX);
```

Comment:
You forgot to change the name of the tested function: getStopPosition => positionToPosX
```suggestion
    const x = positionToPosX(position.start, maxPosition, width, ratioX);
```